src/aca_main.cpp

```diff
@@ -52,6 +52,8 @@ string g_grpc_server_port = EMPTY_STRING;
 string g_ofctl_command = EMPTY_STRING;
 string g_ofctl_target = EMPTY_STRING;
 string g_ofctl_options = EMPTY_STRING;
+string NCM_ADDRESS = EMPTY_STRING;
+string NCM_PORT = EMPTY_STRING;
```

Comment:
for global variables, please use the convention similar to above: g_*

Reply:
Sure, I thought the `g_` means something like google and I was confused. Will change in the next commit.